draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -150,37 +154,39 @@ const Tooltip = ({
   }, [portalSelectorElement, portalSelector])
 
   return (
-    <>
-      <div
-        ref={setReferenceElement}
-        className={classnames(classNameAndIHaveSpokenToDST, {
-          [styles.displayInline]: displayToUse === "inline",
-          [styles.displayBlock]: displayToUse === "block",
-          [styles.displayInlineBlock]: displayToUse === "inline-block",
-          [styles.displayFlex]: displayToUse === "flex",
-          [styles.displayInlineFlex]: displayToUse === "inline-flex",
-        })}
-        onMouseEnter={() => {
-          setIsHover(true)
-        }}
-        onMouseLeave={() => {
-          setIsHover(false)
-        }}
-        onFocusCapture={() => {
-          setIsFocus(true)
-        }}
-        onBlurCapture={() => {
-          setIsFocus(false)
-        }}
-        aria-describedby={tooltipId}
-      >
-        {children}
-      </div>
+    <AnimationProvider isVisible={isHover || isFocus}>
```

Comment:
I don't understand what `AnimationProvider` is doing for us, or what the changes to `AnimationProvider` above are for. I think I'll need someone to explain this to me. 🙏